build_shared.sh

```diff
@@ -65,11 +65,15 @@ for platform in $PLATFORMS; do \
                      >"$KERNEL_TMP"/build.log 2>&1;
 
                 echo "Copying new kernel image ..."
-                cp "$KERNEL_TMP/arch/arm64/boot/Image.gz-dtb" "$KERNEL_TOP/common-kernel/kernel-dtb-$device"
```

Comment:
Please keep this functionality, just like you kept in after your previous PR:
https://github.com/sonyxperiadev/kernel-sony-msm-4.14-common/pull/13/files#diff-89e956a109db67b268e214af2a60a82910b53af0f68793f3e3116e4b7c19e8a0R68

I'll explain below.